debian/wott-agent-self-update.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+apt-get update
+apt-get install wott-agent
```

Comment:
This won't work. It needs the `-y` flag, otherwise it will just throw a dialogue. Change it to `apt-get install -ys wott-agent`